### ISSUE TITLE
Show World/Session Population Limits in World Dash Menu

### ISSUE THREAD

**Issue by FlameSoulis** [Neos-Metaverse/NeosPublic#2585]
## Is your tweak request related to a problem? Please describe.
When viewing worlds with multiple sessions, there is no way to know how many are allowed in the session without first trying to join it, resulting in the "Occupancy Limit Reached" warning, which appears in 3D space and can be obscured by nearby world objects. Likewise, you may not know if anyone is in a world instance unless you spin the 360 previews, which may still not fully show if anyone is in it.

![github](https://user-images.githubusercontent.com/6835457/124369256-d2083f80-dc37-11eb-952a-df175d9c58d4.jpg)

## Describe what would you like tweaked
Simply put: just put the current population number plus the maximum occupancy limit for the given session in the preview before you focus on it.

## Describe alternatives you've considered
The only way to find out if a session is full or not is trial and error, which isn't very user-friendly.

**Comment by Frooxius** [comment]
You can actually click the blue button in the upper right corner to get expanded view, which includes the number of users on the session, session limits and other information. If you press the small arrow in the upper left, you'll get expanded list of users in the session in the small preview as well.

We have some tweaks and additions planned to make it more obvious that those are buttons that you can interact with (e.g. tooltips and some visual changes).

We could potentially add the user maximum to the small session preview as well, but the concern there is that it'll become a bit too cluttered with too much information in small space and become too overwhelming to new users, as that was relatively frequent complaint with old UI.

Note that the message about joining failing is going to be updated as well to be more visible and avoid getting obscured in the environment.
